src/main/java/org/zalando/pazuzu/config/OAuthConfiguration.java

```diff
@@ -17,34 +19,74 @@
 import org.zalando.stups.oauth2.spring.server.TokenInfoResourceServerTokenServices;
 import org.zalando.stups.tokens.config.AccessTokensBeanProperties;
 
+import java.util.Iterator;
+import java.util.List;
+
 @Configuration
 @EnableResourceServer
-@Profile(value = "production")
+@Profile(value = {"production", "test_oauth"})
 public class OAuthConfiguration extends ResourceServerConfigurerAdapter {
 
     @Autowired
     private AccessTokensBeanProperties accessTokensBeanProperties;
 
+    private static final String scopeMatcher = "#oauth2.hasScope('uid')";
+    private String adminMatcher;
+
+    @Value("#{'${pazuzu-registry.admins}'.split(',')}")
+    private List<String> adminList;
+
     @Override
     public void configure(ResourceServerSecurityConfigurer resources) throws Exception {
         // here is the important part for stups-expression-handler
         resources.expressionHandler(new ExtendedOAuth2WebSecurityExpressionHandler());
     }
 
+    private String buildAdminUserMatcher() {
+        Iterator<String> iterator = adminList.iterator();
```

Comment:
But we need to wrap each admin in `authentication.name.equals()`. Do you prefer if I do that in a separate step and then do a join in the end? IMO that won't be much better (and it will be *slower*) :)